test-d/mutable.ts

```diff
@@ -0,0 +1,11 @@
+import {expectType} from 'tsd';
+import {Mutable} from '..';
+
+type Foo = {
+	readonly a: number;
+	readonly b: string;
+};
+
+const ab: Mutable<Foo> = {a: 1, b: '2'};
+expectType<{a: number; b: string}>(ab);
+expectType<Mutable<Readonly<Foo>>>(ab);
```

Comment:
Same as above, change this to:
```suggestion
const ab2 = Mutable<Readonly<Foo>> = ab;
ab2.a = 2;
```